gui/src/main/java/io/bisq/gui/util/GUIUtil.java

```diff
@@ -446,4 +444,22 @@ public static void restoreSeedWords(DeterministicSeed seed, WalletsManager walle
                             .show();
                 }));
     }
+
+    public static void showSelectableTextModal(String title, String text) {
+        TextArea textArea = new TextArea();
+        textArea.setText(text);
+        textArea.setEditable(false);
+        textArea.setWrapText(true);
+        textArea.setPrefSize(800, 600);
+
+        Scene scene = new Scene(textArea);
+        Stage stage = new Stage();
```

Comment:
Most of this code comes from ContractWindow.java.
That class also sets owner (I'm not sure what that is for).
https://github.com/bisq-network/exchange/blob/master/gui/src/main/java/io/bisq/gui/main/overlays/windows/ContractWindow.java#L205